### ISSUE TITLE
Include the class information in the training

### ISSUE THREAD

**Issue by kristosh** [keras-team/keras-contrib#250]
I am using that implementation of wasserstein gans using my database which contain several classes. How can I include that information in the loss function firstly of the discriminator and then in the generator in order to be able to generate images from a specific class?
